src/lib/model/OutboundTransfersModel.js

```diff
@@ -750,7 +752,7 @@ class OutboundTransfersModel {
                     // next transition is to requestQuote
                     await this.stateMachine.requestQuote();
                     this._logger.log(`Quote received for transfer ${this.data.transferId}`);
-                    if(this.stateMachine.state === 'quoteReceived' && !this._autoAcceptQuotes) {
+                    if (this.stateMachine.state === 'quoteReceived' && !this._autoAcceptQuotes) {
```

Comment:
please dont change whitespace unless there is a very good reason to do so.

Reply:
Deleted